lib/rubocop/server/cli.rb

```diff
@@ -23,15 +23,21 @@ class CLI
       STATUS_ERROR = 2
 
       SERVER_OPTIONS = %w[
-        --server --no-server --server-status --restart-server --start-server --stop-server
+        --server
+        --no-server
+        --server-status
+        --restart-server
+        --start-server
+        --stop-server
+        --no-daemonize
       ].freeze
       EXCLUSIVE_OPTIONS = (SERVER_OPTIONS - %w[--server --no-server]).freeze
+      NO_DAEMONIZE_OPTIONS = %w[--server --start-server].freeze
```

Comment:
What do you think about `--restart-server` here? Example, if you had started it daemonized and wanted to restart it in foreground?

Also, should we consider what the output should be if it's not daemonized? 

Should it be `--debug` output?
Should it show simple messages like "processing 5510 files" every time Rubocop is run?

Reply:
Thanks for the feedback! I've made an update to allow `--no-daemonize` to be used with `--restart-server`.

I don't have a very strong opinion about the process output. That said, with the new year starting I have less time to polish this PR up so I could certainly use an assist or strong point in the right direction on how to make the changes you're looking for if you'd be so kind. Thank you!